# LocalizedCell
This explores representation of RNA  abundances in connected parts of a cell:
1. Using transition matrix multiplication, and Newton's method for differential equations for nuclear abundances.
    1.1  For general "RNA" that are pseudorandom sequences
    1.2  For general "micro-RNA" that are constructed by a Monte Carlo method
3. As in 1 except nuclear abundances are determined by a Boolean network (similarly employing Newton's method as in 1).
4. Using an acyclic adjacency list where edge weights are a function of motifs and current abundances.

Transcription and degradation rates are pseudorandom values in ranges determined by numbers from B10NUMB3R5 (Bionumbers); miRNA processing rates are arbitrary at this point.

#### TODO
Make a subclass of Compartments representing actual cellular compartments
Fit numbers to biological data from human, mouse, and plant cells
Generate motif lists from known motifs of RNA binding proteins (RBP)
Plug transcript data into model
Fork and integrate MC Boomer Boolean models of arbitrary pathways
Fork and implement Factory design pattern representation of single cell experiments
Write a more descriptive main.ipynb (initial thinking: represent differential outcomes (e.g. total miRNA knockdown of RNA over a number of steps) as a function of co-localization and compartment-specific coefficients of miRNA and RNA)

For another project: "micro-RNA" that are generated by a Monte Carlo method subject to secondary structure prediction
  might be an illustration of the magnitude of the sample space of possible micro-RNA sequences; but, this could be plugged into LocalizedCell, also.
